lib/zip/file.rb

```diff
@@ -362,6 +362,7 @@ def commit
 
     # Write buffer write changes to buffer and return
     def write_buffer(io = ::StringIO.new(''))
+      io.binmode if io.respond_to?(:binmode)
```

Comment:
The tests still pass without this line. I think that given that the first thing it does is call `::Zip::OutputStream.write_buffer`, which calls `binmode`, this line can be deleted.

Reply:
Yes, you're right. I'll remove this line.